corehq/apps/registry/models.py

```diff
@@ -149,6 +152,9 @@ class RegistryInvitation(models.Model):
     class Meta:
         unique_together = ("registry", "domain")
 
+    def __repr__(self):
+        return f"RegistryInvitation(domain='{self.domain}', status='{self.status}')"
```

Comment:
I left out the registry from this repr and the others to avoid a lookup, but I'm open to feedback on that.

Reply:
you could put the ID in if you like which won't do a lookup